### PR TITLE
Feature/student quiz view

### PR DESCRIPTION
**Pull request by Karolk99**
jestem dobrej myśli

### CODE REVIEW

File: server/StudentList.ts [hunk 1]
```diff
@@ -37,5 +37,9 @@ class StudentList extends EventEmitter{
         return new StudentList();
     }
 
+    compareTwoStudents(student1: Student, student2 : Student) : boolean {
+        return (student1.name == student2.name && student1.surname == student2.surname && student1.nick == student2.nick)
```
Comment (ostatni5) [inline server/StudentList.ts:41]: use `===` not `==`

File: server/controllers/lectures.ts [hunk 2]
```diff
@@ -105,13 +105,27 @@ const addStudentToLecture = async ({
             jsonData["surname"],
             selectedLecture
         );
+        let add = true
+        const array = selectedLecture.studentList.asArray()
```
Comment (ostatni5) [inline server/controllers/lectures.ts:109]: end lines with `;`

File: src/lecturer/components/studentListView/StudentListView.tsx [hunk 3]
```diff
@@ -85,31 +93,42 @@ export function StudentListView(props: StudentListViewProps) {
...
-                            return (
-                                <TableRow key={row.id} className={classes.row}>
-                                    <TableCell>{row.orderIndex}</TableCell>
-                                    <TableCell>{row.nick}</TableCell>
-                                    <TableCell>{row.name}</TableCell>
-                                    <TableCell>{row.surname}</TableCell>
-                                </TableRow>
-                            );
-                        })}
+        <>
```
Comment (ostatni5) [inline src/lecturer/components/studentListView/StudentListView.tsx:102]: remove `<>`

**Review by Derstilon — APPROVED** [review]
There is still work to be done but this is fine for now to merdge

File: src/student/components/quizView/QuizView.tsx [hunk 4]
```diff
@@ -1,21 +1,149 @@
...
+				{ answerText: '7', isCorrect: true },
+			],
+		},
+	];
+
+    const openQuestions = [
+        {
+			questionText: 'The iPhone was created by which company?',
+		},
+    ];
```
Comment (ostatni5) [inline src/student/components/quizView/QuizView.tsx:71]: move mock data outside component

File: src/student/components/quizView/QuizView.tsx [hunk 5]
```diff
@@ -1,21 +1,149 @@
...
+    }
+    
+    const handleTextAreaChange = ( e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>, questionNumber : number) => {
+        const { value } = e.target;
+        console.log("Open question number: ", questionNumber, "asnwer: ", value);
+    };
+    
+    const submit = () => {
+        console.log("Submit");
+    }
```
Comment (ostatni5) [inline src/student/components/quizView/QuizView.tsx:84]: On submit you should a console log with all quiz answers.

**Review by ostatni5 — COMMENTED** [review]
Please next time use smaller components.

**Pull request closed without merging**